updateCli.d/jenkins-release.yaml

```diff
@@ -0,0 +1,21 @@
+source:
+  kind: dockerDigest
+  spec:
+    image: "jenkins/jenkins"
+    tag: "lts"
```

Comment:
should this be?:
```suggestion
    tag: "lts-jdk11"
```

Reply:
Yes, nice catch then we also have to update the digest